.env.example

```
@@ -0,0 +1,26 @@
+# per-job directories (created and removed automatically per run).
+#
+# The values below are placeholders; replace them with your own machine.
+
+# SSH host name or address of the CUDA machine.
+PHONO_GPU_HOST=192.168.1.100
+
+# SSH user on the host (key-based, no password prompts).
+PHONO_GPU_USER=gpuuser
+
+# Human label used in runner messages and benchmark reports.
+PHONO_GPU_NAME=lab GPU
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)